css/index.css

```diff
@@ -0,0 +1,89 @@
+body {
+  background-color: #90dad8;
+  color: #004640;
+  font-family: "Nunito", sans-serif;
+  text-align: center;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+}
+
+h1 {
+  flex: 1;
+  font-family: Georgia, serif;
+  font-size: 70px;
+  text-align: right;
+  color: #c97796;
+  margin: 5px;
+  padding: 5px;
+  border-bottom: double #c97796 2px;
+  border-top: double #c97796 2px;
+}
+
+#ja-photo {
+  max-width: 30%;
+  padding: 20px;
+  border-radius: 7%;
+  position: relative;
+  align-self: right;
+}
+
+h2 {
+  font-size: 35px;
+  font-weight: bold;
+  color: #ca976c;
+}
+
+#About {
+  background-color: #f6dce6;
+  margin: 25px;
+  padding: 35px;
+  border-radius: 2%;
+}
+
+#Experience {
```

Comment:
I *think* that the conventional style is to name things in an HTML doc using all lowercase, with words separated by hyphens. This works as is, but just wanted to give you a heads up on the typical practice.